frontend/ui/widget/textboxwidget.lua

```diff
@@ -965,7 +982,7 @@ function TextBoxWidget:_renderImage(start_row_idx)
                                 y = self.dimen.y,
                                 w = image.width,
                                 h = image.height,
-                            }
+                            }, true
```

Comment:
If that `true` means "dithered", as it's on a single line, add that as a comment (so I don't have to go look at setDirty :)

Reply:
It does, yeah.